src/brs/http/api/SubscriptionDto.java

```diff
@@ -0,0 +1,28 @@
+package brs.http.api;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@AllArgsConstructor
+@Builder
+public class SubscriptionDto {
+	public String id;
+
+	// TODO sender and senderRS for backward compatibility.
+	// With the type AccountDto there is a new way to provide the data using more reusable code. 
```

Comment:
no need for such comments - code is selfexplaining IMHO

Reply:
done